### PR TITLE
AP_GPS: Send RTK baseline data from SBF

### PR DESCRIPTION
**Pull request by WickedShell**
This supports sending the `GPS_RTK` from Septentrio units. This has been bench tested with a mosaic receiving corrections from a Altus NR2 base. The corrections mirror what you get inside of RxTools when double checking it.

@amilcarlucas do you have a setup to independently test/replicate the results?

### CODE REVIEW

**Review by amilcarlucas — COMMENTED** [review]
All changes look correct to me.
Will test thid this week, I hope.

File: libraries/AP_GPS/AP_GPS_SBF.cpp [hunk 1]
```diff
@@ -411,6 +414,41 @@ AP_GPS_SBF::process_message(void)
         }
         break;
     }
+    case BaseVectorGeod:
+    {
+        const msg4028 &temp = sbf_msg.data.msg4028u;
+
+        // just breakout the const's we need for DNU's
```
[the remark this answers is not in the document]
Comment (WickedShell) [reply libraries/AP_GPS/AP_GPS_SBF.cpp]: Didn't want to change the comment next time, so I've made it more general :)

**Comment by WickedShell** [comment]
I've pushed up a patch that I think addresses all of Peter's comments. I still need to verify the single-precision-constant bit. Regarding the use all or none, I'm inclined to just copy them all in as we have the data, in practice they seem to all change at once so I'm not to worried.

**Comment by tridge** [comment]
@WickedShell I've pushed a change that fixes the build on cygwin

**Pull request merged**